lib/common.js

```diff
@@ -97,6 +100,10 @@ function createConfig(env = {}) {
   const mergeRetries = parsePositiveInt("MERGE_RETRIES", 6);
   const mergeRetrySleep = parsePositiveInt("MERGE_RETRY_SLEEP", 5000);
   const mergeDeleteBranch = env.MERGE_DELETE_BRANCH === "true";
+  const mergeProtectBranches = parseCommaSeparatedList(
```

Comment:
This should be renamed to match the MERGE_DELETE_BRANCH_FILTER name 🙏  (see other variables in lines 90-100)